### PR TITLE
DBP-657 upgrade awx to 2.14.0

### PR DESCRIPTION
**Pull request by JannikBadenhop**
# Description
- Upgrade awx to version **2.15.0** (not 2.14.0, since 2.15 just released and fixes a relevant issue)
- Split kustomize setup into dev and admin to allow for staged rollout
- add small leftover changes to onepassword operator deployment

## Links to Tickets or other pull requests
[DBP-657](https://ticketsystem.dbildungscloud.de/browse/DBP-657)
https://github.com/hpi-schul-cloud/infra-schulcloud/pull/1088

## Approval for review

- [ ] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### CODE REVIEW

**Comment by JannikBadenhop** [comment]
Apparently some very relevant images for awx are missing from quay.io as of now. So i am putting this issue on hold

File: roles/awx/templates/infra/awx-operator.yml [hunk 1]
```diff
@@ -0,0 +1,2853 @@
...
+    served: true
+    storage: true
+    subresources:
+      status: {}
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: awx-operator-controller-manager
+  namespace: awx
```
Comment (simoncolincap) [inline roles/awx/templates/infra/awx-operator.yml:2459]: The namespaces should be set to the Ansible variable ansible_namespace

**Review by simoncolincap — APPROVED** [review]

File: roles/awx/templates/infra/awx-operator.yml (hunk 1 above)
Comment (JannikBadenhop) [reply roles/awx/templates/infra/awx-operator.yml]: thanks, it's included now. i updated the documentation to include a reminder for this

File: roles/awx/tasks/main.yml [hunk 2]
```diff
@@ -61,9 +61,7 @@
             projects_persistence: "{{ awx_projects_persistence }}"
             projects_storage_access_mode: "{{ awx_projects_storage_access_mode }}"
             projects_storage_size: "{{ awx_projects_storage_size }}"
-            postgres_data_volume_init: true
-            postgres_init_container_commands: |
-              chown 26:0 /var/lib/pgsql/data
-              chmod 700 /var/lib/pgsql/data
+            postgres_data_volume_init: "{{ postgres_data_volume_init | default(omit, true) }}"
```
Comment (simoncolincap) [inline roles/awx/tasks/main.yml:64]: Did you check the case where omit is used? It can cause strange results sometimes

Comment (JannikBadenhop) [reply roles/awx/tasks/main.yml:64]: i did, had to trial and error through a few recursive assignments, but it works now with and without the var being set.

**Review by simoncolincap — APPROVED** [review]

**Review by simoncolincap — CHANGES_REQUESTED** [review]
I think the new AWX variables should have awx_ as a prefix to prevent collisions with potentially existing variables

**Pull request closed without merging**